website/docs/ephemeral-resources/cognito_identity_openid_token_for_developer_identity.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Cognito Identity"
+layout: "aws"
+page_title: "AWS: aws_cognito_identity_openid_token_for_developer_identity"
+description: |-
+  Terraform ephemeral resource for managing an AWS Cognito Identity Open ID Token for Developer Identity.
+---
+
+
+# Ephemeral: aws_cognito_identity_openid_token_for_developer_identity
+
+Terraform ephemeral resource for managing an AWS Cognito Identity Open ID Token for Developer Identity.
+
```

Comment:
I believe we typically include a note similar to the following for ephemeral resource docs.

```suggestion

~> Ephemeral resources are a new feature and may evolve as we continue to explore their most effective uses. [Learn more](https://developer.hashicorp.com/terraform/language/v1.10.x/resources/ephemeral).

```